### PR TITLE
Add Laravel Kit 1.1.0

### PR DESCRIPTION
**Pull request by tmdh**
After making all changes to the cask:

- [x] `brew cask audit --download laravel-kit' is error-free.
- [x] `brew cask style --fix laravel-kit' reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install laravel-kit` worked successfully.
- [x] `brew cask uninstall laravel-kit` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/laravel-kit.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'laravel-kit' do
+  version '1.1.0'
+  sha256 :no_check
```
Comment (commitay) [inline Casks/laravel-kit.rb]: Add the `sha256`

File: Casks/laravel-kit.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  url "https://github.com/tarequemdhanif/laravel-kit/releases/download/v#{version}/laravel-kit-#{version}-mac.zip"
+  name 'Laravel Kit'
```
Comment (commitay) [inline Casks/laravel-kit.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

Example:
https://github.com/caskroom/homebrew-cask/blob/0480e1f119f2742586f35b65a4efbfba670757b8/Casks/autodmg.rb#L5-L8

**Review by commitay — CHANGES_REQUESTED** [review]

**Comment by tarequemdhanif** [comment]
How to get the `checkpoint`?

File: Casks/laravel-kit.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+cask 'laravel-kit' do
+  version '1.1.0'
+  sha256 '58e12c1934d1e9d2eb474dab13ba40c282fa3f1e8e20251c5c488e7df1adc56b'
+
+  url "https://github.com/tarequemdhanif/laravel-kit/releases/download/v#{version}/laravel-kit-#{version}-mac.zip"
+  appcast 'https://github.com/tarequemdhanif/laravel-kit/releases.atom'
```
Comment (vitorgalvao) [inline Casks/laravel-kit.rb]: It’s missing the `checkpoint`. See [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Pull request merged**